kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-common/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/common/client/converters/DMNMarshallerImportsHelperKogitoImpl.java

```diff
@@ -16,29 +16,256 @@
+                                        .map(node -> includedModelFactory.makeDMNIncludeNode(path, model, node))
+                                        .collect(Collectors.toList());
+                                result.addAll(nodes);
+                                success.onInvoke(nodes);
+                            }
+
+                            @Override
+                            public void onError(final ClientRuntimeError error) {
+                                LOGGER.log(Level.SEVERE, error.getMessage());
+                                fail.onInvoke(error);
+                            }
+                        });
+                    });
+                });
+    }
+
+    @Override
+    public void loadModels(final ServiceCallback<List<IncludedModel>> callback) {
+        final List<IncludedModel> models = new Vector<>();
+        contentService.getFilteredItems(DMN_FILES_PATTERN, ResourceListOptions.assetFolder())
+                .then(items -> promises.all(Arrays.asList(items), file -> contentService.loadFile(file).then(fileContent -> {
+                    diagramService.transform(fileContent, new ServiceCallback<Diagram>() {
+                        @Override
+                        public void onSuccess(final Diagram item) {
+                            final String modelPackage = "";
+                            final Diagram<Graph, Metadata> diagram = (Diagram<Graph, Metadata>) item;
+                            final String namespace = diagramUtils.getNamespace(diagram);
+                            final String importType = DMNImportTypes.DMN.getDefaultNamespace();
+                            final int drgElementCount = diagramUtils.getDRGElements(diagram).size();
+                            final int itemDefinitionCount = diagramUtils.getDefinitions(diagram).getItemDefinition().size();
+                            final String filename = FileUtils.getFileName(file);
+                            models.add(new DMNIncludedModel(filename,
+                                                            modelPackage,
+                                                            filename,
+                                                            namespace,
+                                                            importType,
+                                                            drgElementCount,
+                                                            itemDefinitionCount));
+                        }
+
+                        @Override
+                        public void onError(final ClientRuntimeError error) {
+                            LOGGER.log(Level.SEVERE, error.getMessage());
+                        }
+                    });
+                    return promises.resolve();
+                })).then(v -> {
+                    callback.onSuccess(models);
+                    return promises.resolve();
+                }));
+    }
+
+    Promise loadDefinitionFromFile(final String file,
```

Comment:
IntelliJ is warning about this method, I think we can solve it by using:
```suggestion
    Promise<Void> loadDefinitionFromFile(final String file,
```